ddtrace/opentracer/tracer.go

```diff
@@ -47,7 +50,9 @@ func (t *opentracer) StartSpan(operationName string, options ...opentracing.Star
 	}
 	opts := []ddtrace.StartSpanOption{tracer.StartTime(sso.StartTime)}
 	for _, ref := range sso.References {
-		if v, ok := ref.ReferencedContext.(ddtrace.SpanContext); ok && ref.Type == opentracing.ChildOfRef {
+		// opentracing.ChildOfRef and opentracing.FollowsFromRef will be represented as children
+		// Datadog tracing UI does not have a concept of Follows From references
```

Comment:
```suggestion
		// Datadog APM does not have a concept of FollowsFrom references
```
Please put the comment inside the `if` statement. It's a coding style we've been using in this repository: if the comment refers to a condition, that comment should be inside the condition.